README.md

```diff
@@ -18,7 +18,7 @@ But if you came here because you want to contribute to DreamBerd, here are some
 > **New!** Read the Dreamberd release notes [here](https://github.com/TodePond/DreamBerd/releases/tag/v.release.notes).
 
 [<img align="right" height="100" src="dreamberd.svg">](https://github.com/TodePond/DreamBerd/blob/main/examples/Examples.md "Click here for the examples page.")
-
+ 
```

Comment:
Is this space intended?

Reply:
Doesn't have any UI effect. Its for readability

Reply:
I don't like trailing spaces